libdrizzle/drizzle_local.h

```diff
@@ -69,11 +69,52 @@ void drizzle_set_error(drizzle_st *con, const char *function, const char *format
  */
 void drizzle_free(drizzle_st *con);
 
+/* Create a copy of drizzle_st instance
+ *
+ * @params[out] drizzle the target instance
+ * @params[in] from the instance to clone
+ * @return a newly allocated and configured copy of a drizzle connection
+ */
 drizzle_st *drizzle_clone(drizzle_st *drizzle, const drizzle_st *from);
 
+/*
+ * Initializing the shared ssl object, by calling pthread_once
```

Comment:
Maybe `Initialize` to be consistent with the other functions/methods documentation?